semantic/lookup.go

```diff
@@ -1,26 +1,26 @@
 package semantic
 
 import (
+	"fmt"
 	flatbuffers "github.com/google/flatbuffers/go"
 	"github.com/influxdata/flux/codes"
 	"github.com/influxdata/flux/internal/errors"
 	"github.com/influxdata/flux/libflux/go/libflux"
 	"github.com/influxdata/flux/semantic/internal/fbsemantic"
 )
 
+var byteArr = libflux.EnvStdlib()
```

Comment:
Global variables should be used very sparingly, so in this situation it would be good to only create one of them.  I would nest the calls here, like this
```
var environment = TypeEnvMap(fbsemantic.GetRootAsTypeEnvironment(libflux.EnvStdlib(), 0))
```
It's harder to read, but I think that fewer globals is still better.

The name `environment` will be visible everywhere in the semantic package, so it would be good to rename it to something that's unlikely to clash, like `stdlibTypeEnvironment` or something.